.github/workflows/test.yaml

```diff
@@ -0,0 +1,21 @@
+---
+name: Lint
```

Comment:
should be changed to Test

Reply:
Good catch @JefeDavis . I’m away from my computer for a few days. Any way I can convince you to push the change up so we can get it merged?